### ISSUE TITLE
Improve display of installed variants

### ISSUE THREAD

**Issue by yan12125** [macports/macports-webapp#53]
Homebrew collects the number of installations with various options. For example, in https://formulae.brew.sh/formula/wget, there are `wget`, `wget --HEAD`, `wget --with-debug` and so on. Seems there is nothing similar in https://frozen-falls-98471.herokuapp.com/port/wget/?

Homebrew developers use such statistics to determine what kind of options should be the default. I believe port maintainers can enjoy similar benefits from such statistics, too.

**Comment by jmroot** [comment]
Variant information is already collected, it just needs to be displayed.

**Comment by yan12125** [comment]
Great to know that!

**Comment by arjunsalyan** [comment]
Thank you. Variants shall be soon added.

https://frozen-falls-98471.herokuapp.com/port/wget/ is now outdated. The latest version runs on ec2 now:

http://ec2-52-34-234-111.us-west-2.compute.amazonaws.com

**Comment by yan12125** [comment]
Thanks for clarification! Mind to update the link for this GitHub repository?

![圖片](https://user-images.githubusercontent.com/1937689/62222743-721f1380-b3e6-11e9-821c-564d147f7533.png)

**Comment by mojca** [comment]
Done, thanks. In any case we hope to be able to deploy it soon.

**Comment by yan12125** [comment]
Thanks for the update! Here are some ideas:

![圖片](https://user-images.githubusercontent.com/1937689/63141856-e9c18500-c019-11e9-9f89-777b764a0b28.png)
(from https://ports.macports.org/port/wget/?tab=stats&days=30&days_ago=0)

* Does the first row indicate wget without any variants? If so, it might be better to write something like "(no variants)" in the first row.
* `+gnutls`, `+ssl` might be better than `gnutls +`, `ssl +`

**Comment by mojca** [comment]
I agree with both and also wrote both down to the [long document with minor pending issues](https://docs.google.com/document/d/1TqNHm3NrtLDelWqY7P4pZyTMZDJrlsPymFZklzV2t24/edit#heading=h.5ust9cmqybh1).

Yes, the trailing `+` should go.

Quoting myself on the other point:
> For variants we should distinguish between `NULL` (variants unknown) and in that case write `(unknown)` and an empty string and in that case write `(none)`. If no variants are the most popular case, also make sure that `(none)` is written in the above suggested table entry. We should not count unknown (`NULL`) values to determine the most popular variants.

The empty field doesn't necessarily mean "no variants", but mostly "no data", for the following reasons:
* we didn't have the required field in the table until you opened this issue
* we did not record the raw submission data during a certain period of time
* we do have some raw data from before we introduced another field, but we did not run the sql query which would extract those data (for cases of existing raw data and missing variants)
* `NULL` values are currently not properly distinguished from empty strings.

We need to fix the issues and probably rerun the query to fill in (fix) the data from raw submissions (for every single submission where raw data existing; where it doesn't, make sure that `NULL` gets filled).

**Comment by yan12125** [comment]
Thanks for clarification! By the way, the document for pending issues is awesome.
